test/tests/functional/pbs_multi_sched.py

```diff
@@ -2059,24 +2058,12 @@ def test_setting_default_partition(self):
 
         self.common_setup()
         a = {'partition': 'pbs-default'}
-        try:
+        with self.assertRaises(PbsManagerError) as ex:
```

Comment:
Suggestion: be consistent with your names of exceptions.  You used 'e' above, and 'ex' here.

Reply:
Thanks, I've changed its name. Please have a look.